examples/apps/cli/main.c

```diff
@@ -97,3 +98,18 @@ int main(int argc, char *argv[])
 
     return 0;
 }
+
+/*
+ * Provide a "weak" log function the platform code can override if desired.
+ */
+#if !OPENTHREAD_ENABLE_DEBUG_UART_LOG
+OT_TOOL_WEAK
+void otPlatLog(otLogLevel aLogLevel, otLogRegion aLogRegion, const char *aFormat, ...)
+{
+    va_list ap;
+
+    va_start(ap, aFormat);
+    otCliPlatLogv(aLogLevel, aLogRegion, aFormat, ap);
+    va_end(ap);
+}
```

Comment:
BTW - please let me know before I make this change as it is going to effect a few platforms that define their own logging scheme.